docs/20_smart-contracts/10_getting-started/20_accounts/index.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Account Basics
+---
+
+## What is an account?
+
+An account on the EOS Network is exactly what it sounds like. It’s a digital container that holds a variety of information about you. It can even hold a smart contract. It is the key component that enables you to access and control your blockchain data, as well as execute transactions.
+
+
+## Account names
+
+Every account on the EOS Network has a human-readable name. This makes it easier to identify transaction recipients and smart contracts. In order to keep account names efficient on the blockchain, a few restrictions apply to all names: 
+
+
+
+* All characters must be lowercase
+* Every name must be 12 characters long (or less with a suffix/premium name)
+* Only letters `a-z`, numbers `1-5`, and period (`.`) are supported characters. 
+* Names cannot start with a number or a period. 
+* Names cannot end with a period.
+
+Periods have a special meaning for EOS accounts. They specify that an account has a **suffix** (similar to a top-level domain like .com), also known as a **premium name**. Accounts with a suffix can only be created by the **suffix owner**. 
+
+For instance, if someone owns the suffix `.bar` then only that person can create `foo.bar`. 
+
+At the end of this guide, we will talk about the Name Bidding system which allows someone on the EOS network to purchase premium names. 
+
+
+## Public/Private Keys
+
+Every EOS account is ultimately controlled by a public/private key pair. The private key is used to sign transactions and must be kept confidential, while the public key is used to identify the account on the blockchain and can be publicly known.
+
+It is important to safeguard your private key, as it provides complete control over your account's digital assets. Your private key should be stored in a secure software or hardware wallet, as whoever possesses it will have full access to the assets linked to your account.
+
+EOS accounts offer extra security mechanisms out of the box, using something called the *permission system*.
+
+Examples of private and public keys: 
+```
+Private: 5KSdyAiFzYQAtBKDBKCCF28KMMhZ4EmXUxSg8B3nSkHKutT15rY
+Public: PUB_K1_5d7eRKgCCiEdsbBdxxnZdFWnGYS64uWZPZgTcTU1xnB2aESxqR
+Legacy Public Format: EOS5d7eRKgCCiEdsbBdxxnZdFWnGYS64uWZPZgTcTU1xnB2cq4JMD
+```
+
+
+## Permissions system
+
+Each account has a set of hierarchical permissions that control it, and comes with two base permissions by default. These two permissions cannot be removed as they are required for your account to function properly. 

```

Comment:
I would replace "control it" with "control what the account can do". This hints that permissions are critical for determining what actions the account can execute w/o defining actions, authorizations, threshold, etc.